File: table.java
import java.util.Scanner;
class table{
    public static void main(String args[])
    {
        Scanner scn=new Scanner(System.in);
        System.out.println("enter which table to be printd");
        int n=scn.nextInt();
          
    for(int i=1;i<=10;i++){
        System.out.println(i*n);
    }
    }
}

File: table.cpp
#include <iostream>
using namespace std;
int main(int args,char** argb){
    int i,n;
    cout<<"enter which table to be print "<<endl;
    cin>>n;
    for(i =1;i<=10;i++){
        int k=i*n;
        cout<<k<<endl;
    }
}
